### PR TITLE
New crud commands

### LINKED ISSUE #3: New CRUD commands
`/notelist` – Display a list of all files or notes. Format:
```
1) {note name} - {id}
2) {note name} - {id}
```
id must be copyable. Use `mono` formatting

`/getfile` – Retrieve a file by passing its identifier. "Введіть id файлу який ви хочете отримати:"
`/editnote` – Edit an existing note or file. First of all receive id of file to be edited. Then fetch it's current fields, than let user choose what he wants to change via buttons. He should have an ability to change all fields.
`/deletenote` - "Введіть id файлу який ви хочете видалити:". Receive id and send request to api to delete file.

API - https://sr-kpi-api-development.up.railway.app/api#/Documents

### PR DESCRIPTION
**Pull request by MrPaschenko**
Closes #3

### CODE REVIEW

File: src/bot.ts [hunk 1]
```diff
@@ -4,60 +4,232 @@ import { Bot, Context, InputFile, InlineKeyboard } from "grammy";
...
+
+    try {
+        const response = await instance.get("/documents/notes");
+
+        if (Array.isArray(response.data) && response.data.length === 0) {
+            await ctx.reply("Записів нема");
+        } else {
+            const formattedData = response.data
+                .map((item, index) => `${index + 1}\\) ${item.name.replaceAll(".", "\\.").replaceAll("_", "\\_")} \\- \`${item.id}\``)
+                .join("\n");
```
Comment (Markusplay) [inline src/bot.ts:65]: extract data formatting to the separated function, your logic is duplicating

File: src/bot.ts [hunk 2]
```diff
@@ -4,60 +4,232 @@ import { Bot, Context, InputFile, InlineKeyboard } from "grammy";
...
+
+//сховище для стоврення та зміни записки
 const memoData = {};
 
+// id для зміни файлу
+let editFileId = 0;
+
+const instance = axios.create({
+    baseURL: "https://sr-kpi-api-development.up.railway.app",
+});
```
Comment (Markusplay) [inline src/bot.ts:19]: instance could be reused in toher parts of project. Move it to separate file

**Review by Markusplay — CHANGES_REQUESTED** [review]

File: src/bot.ts (hunk 1 above)
Comment (MrPaschenko) [reply src/bot.ts]: Fixed

File: src/bot.ts (hunk 2 above)
Comment (MrPaschenko) [reply src/bot.ts]: Fixed

**Review by Markusplay — APPROVED** [review]

**Pull request merged**